### ISSUE TITLE
Failed to open music pack directory.

### ISSUE THREAD

**Issue by DavideCanno** [fabiangreffrath/crispy-doom#768]
### Background

Version of Crispy Doom: 5.10.3

Operating System and version: Windows 10 21H1

Game: (Doom/Heretic/Hexen/Strife/other) DOOM II

Any loaded WADs and mods (please include full command line): doom2.wad

### Bug description

Observed behavior: When I extracted the music pack for DOOM II: Hell on Earth, when I went to Cripsy Doom Setup and clicked on "[M] Music Packs", it said "Failed to open music pack directory.". Weirdly I looked at AppData after booting DOOM II and it created the directories, I copied the files there but still didn't work.

Expected behavior: Opening File Explorer on the directory for music

**Comment by fabiangreffrath** [comment]
Does your appdata directory name contain spaces or non-latin characters?

**Comment by DavideCanno** [comment]
no, its simply C:\Users\david\AppData\Roaming (i just copied the path from Explorer)
